dist/js/eslint.test.js

```diff
@@ -0,0 +1,11 @@
+function eslintFunctionTest() {
+  return true;
+}
+
+eslintTest();
+var eslintObjectTest = function () {
```

Comment:
Set your eslint configuration to flag out the use of 'var'. You should use es6 'let' or 'const' keyword for variable declaration. Also make use of es6 arrow function for function declaration